examples/transfer/plot_rpa_steps.py

```diff
@@ -0,0 +1,109 @@
+"""
+====================================================================
+Plot the data transformations in the Riemannian Procrustes Analysis
+====================================================================
+
+Use the SpectralEmbedding module to plot in 2D the transformations on the data
+points from source and target domains when applying the Riemannian Procrustes
+Analysis [1] to match their statistics.
+
+.. [1] PLC Rodrigues et al “Riemannian Procrustes analysis: transfer learning
+for brain-computer interfaces”. IEEE Transactions on Biomedical Engineering,
+vol. 66, no. 8, pp. 2390-2401, December, 2018
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+
+from pyriemann.embedding import SpectralEmbedding
+from pyriemann.datasets.simulated import make_example_transfer_learning
+
+from pyriemann.transfer import (
+    decode_domains,
+    TLCenter,
+    TLRotate,
+)
+
+
+# fix seed for reproducible results
+seed = 66
+
+# create source and target datasets
+N = 50
+X_enc, y_enc = make_example_transfer_learning(
+    N=N,
```

Comment:
I don’t like much the N parameter. It should n_matrices ? I don’t remember the naming we agreed

Reply:
Yeah, it's already corrected to `n_matrices`